src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -94,20 +91,38 @@ public IntakeMap getIntakeMap() {
 
     @Override
     public ClimberMap getLeftClimberMap() {
-        final DigitalInput leftUpperLimit = new DigitalInput(1);
-        final DigitalInput leftLowerLimit = new DigitalInput(2);
-        final PIDSparkMax leftMotor = new PIDSparkMax(9, MotorType.kBrushless);
-        leftMotor.validateCurrent(CLIMBER_CURRENT_LIMIT);
-        return new ClimberMap(leftMotor, leftUpperLimit::get, leftLowerLimit::get);
+        // The current limit for the climber's motors in amps
+        final int EXTEND_LIMIT = 20;
+        final int ROTATE_LIMIT = 20;
+
+        final PIDSparkMax extendMotor = new PIDSparkMax(9, MotorType.kBrushless);
+
+        final PIDSparkMax rotateMotor = new PIDSparkMax(18, MotorType.kBrushless);
+
+        // Setting the current limits on both the validators and motor controllers
+        extendMotor.validateCurrent(EXTEND_LIMIT);
+        extendMotor.getMotorController().setSmartCurrentLimit(EXTEND_LIMIT);
+        rotateMotor.validateCurrent(ROTATE_LIMIT);
+        rotateMotor.getMotorController().setSmartCurrentLimit(ROTATE_LIMIT);
+
+        return new ClimberMap(extendMotor, rotateMotor);
     }
 
     @Override
     public ClimberMap getRightClimberMap() {
-        final DigitalInput rightUpperLimit = new DigitalInput(3);
-        final DigitalInput rightLowerLimit = new DigitalInput(4);
-        final PIDSparkMax rightMotor = new PIDSparkMax(10, MotorType.kBrushless);
-        rightMotor.validateCurrent(CLIMBER_CURRENT_LIMIT);
+        // The current limit for the climber's motors in amps
+        final int EXTEND_LIMIT = 20;
```

Comment:
Hmm we probably shouldn't define these twice. I think here we need to put these outside the function and share them.